recipes/cad1/task1/baseline/README.md

```diff
@@ -0,0 +1,148 @@
+```bash
+tar -xvzf <PACKAGE_NAME>
+```
+
+### 1.1 Necessary data
+* **Music** contains the MUSDB18-HQ music dataset for training, validation and evaluation.
+```text
+cadenza_data
+└───task1
+    └───audio
+        └───musdb18hq
+            ├───train
+            └───test
+```
+* **Metadata** contains the metadata for the systems.
+```text
+cadenza_data
+└───task1
+    └───metadata
+        └───musdb18hq
+            ├───listeners.json
+            ├───musdb18_train.json
+            ├───musdb18_valid.json
+            └───musdb18_test.json
+```
+
+### 1.2 Additional optional data
+* **MedleyDB** contains both MedleyDB versions 1 [4] and 2 [5] datasets.
+MedleyDB dataset is not used in the evaluation and is your responsibility to exclude any song that is already in the training set.
+```text
+cadenza_data
+└───task1
+    └───audio
+        └───MedleyDB
+            ├───Audio
+            └───Metadata
+```
+
+* **BACH10** contains the BACH10 dataset [6].
+BACH10 dataset is not used in the evaluation.
+```text
+cadenza_data
+└───task1
+    └───audio
+        └───fma_small
+            ├───000
+            ├───001
+            ├───...
+```
+
+* **FMA Small** contains the FMA small subset of the FMA dataset [7].
+FMA small dataset is not used in the evaluation. This dataset does not provide the independent stems, but only the full mix.
+However, it can be used for any unsupervised learning task.
+```text
+cadenza_data
+└───task1
+    └───audio
+        └───fma_small
+            ├───000
+            ├───001
+            ├───...
+```
+
+## 2. Baseline
+
+In the `baseline/` folder, we provide code for running the baseline enhancement system and performing the objective evaluation.
+Note that we use hydra for config handling. For more information, see [here](https://hydra.cc/docs/intro/).
+
+### 2.1 Enhancement
+
+The baseline enhance simply takes the out-of-the-box [Hybrid Demucs](https://github.com/facebookresearch/demucs)[1]
+source separation model distributed on [TorchAudio](https://pytorch.org/audio/main/tutorials/hybrid_demucs_tutorial.html)
+and applies a simple NAL-R [2] fitting amplification and a simple automatic gain compressor to each
+VDBO (`vocals`, `drums`, `bass` and `others`) stem.
+
+The remixing is performed by summing the amplified VDBO stems.
+
+The baseline will generate a left and right signal for each VDBO stem and a remixed signal, totalling 9 signals per song-listener.
+
+To run the baseline enhancement system, firstly specify `paths.root` in `config.yaml` to point to
+where you have installed the clarity data. You can also define your own `path.exp_folder` to store enhanced
+signals and evaluated results.
+Then run:
+```bash
+python enhance.py
+```
+Alternatively, you can provide the root variable on the command line, e.g.,
+```bash
+python enhance.py path.root=/full/path/to/my/cadenza_data
+```
+To get a full list of the parameters, run:
+```bash
+python enhance.py --help
```

Comment:
Not for now, but we can look at setting up an [entry-point](https://packaging.python.org/en/latest/specifications/entry-points/) in future revisions to simplify invoking the commands. Apologies for not clocking/suggesting this earlier.

Reply:
We are using hydra. Hydra works using both, command line parameters like argparse and default parameters using the config.yaml

I am not sure we need entry-point for the baseline systems. maybe for the rest of the toolkit